packages/dev/docs/pages/releases/2022-05-27.mdx

```diff
@@ -0,0 +1,49 @@
+date: 2022-05-27
+---
+
+# May 27, 2022 Release
+
+Ever since React 18 was announced last July, compatibility has been front of mind for the team. We are extremely pleased to announce that React 18 is now officially supported with our library! Special thanks to [@eps1lon](https://github.com/eps1lon) for his
+help with getting our tests working with React 18.
+
+A major version bump to our workflow icons is included in this release so be sure to upgrade accordingly. Other updates include added support for emphasized Tabs and various bug fixes to our existing components and hooks. Thank you to all who contributed to this release!
+
+## New features
+- Support for isEmphasized Tabs - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/3124)
+
+## Updates
+- React 18 support - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/3108)
+- Update dom-helpers - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3110)
+- Upgrade workflow icons - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3019)
+
+## Fixes
+- Reduce the number of rerenders from focus hooks - [@csantos1113](https://github.com/csantos1113) - [PR](https://github.com/adobe/react-spectrum/pull/3045)
+- Fix NumberParser mistakenly returning Arabic values in non-Arabic locales  - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3116)
+- Add ActionMenu to mono-package - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3144)
+- Distinguish Talkback virtual clicks from Windows FF touch event - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/3136)
+- Fixes missing 'props' from API sections of useFocus and useFocusWithin docs - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/3161)
+- Fix ButtonGroup visual flicker on resize check - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/3160)
+- Fix React error for Tabs when every key is disabled - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3162)
+
+## Under construction
+
+Fixes to pre-released components are listed below. Please feel free to try them out, and report any issues you encounter.
+
+- Fix edge cases in DatePicker with min/max dates and eras - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/3149)
+- Make DatePicker button keyboard focusable - [@majornista](https://github.com/majornista) - [PR](https://github.com/adobe/react-spectrum/pull/3145)
```

Comment:
Do we wanna call these out?

Reply:
Maybe just in the Slack annoucement?

Reply:
Maybe? There is precedent for announcing updates to pre-release components though so I'm not sure

Reply:
usually I just put the pre-release packages that were updated and their docs.  I guess in this case it would be all of them because of the react 18 change

Reply:
right, I'm assuming we don't wanna list them all. The precedent I'm referring to is https://react-spectrum.adobe.com/releases/2022-03-21.html though where we announced specific pre-release component changes